app/presenters/deploy_group_usage_csv_presenter.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+require 'csv'
+
+class DeployGroupUsageCsvPresenter
+  def self.to_csv
```

Comment:
use `class << self` and then `def to_csv` and all the other methods can be in `private`

Reply:
done